Scripts/Core/VRView.cs

```diff
@@ -189,6 +184,70 @@ public void OnEnable()
             m_CameraRig.position = headCenteredOrigin;
             m_CameraRig.rotation = Quaternion.identity;
 
+            if (s_ExistingSceneMainCamera)
+            {
+                var cameraGameObject = m_Camera.gameObject;
+                var potentialImageEffects = s_ExistingSceneMainCamera.GetComponents<MonoBehaviour>();
+                var enabledPotentialImageEffects = potentialImageEffects.Where(x => x != null && x.enabled);
+                var targetMethodNames = new [] {"OnRenderImage", "OnPreRender", "OnPostRender"};
+                var bindingFlags = BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.Static;
+                foreach (var potentialImageEffect in enabledPotentialImageEffects)
+                {
+                    var componentInstanceType = potentialImageEffect.GetType();
+                    var targetMethodFound = false;
+                    for (int i = 0; i < targetMethodNames.Length; ++i)
+                    {
+                        // Each of the three checks is performed to catch the various image effect variants I've tested against
+                        // Each check catches a different case that was encountered during testing
+                        // Check isntanced type for target methods
+                        targetMethodFound = componentInstanceType.GetMethod(targetMethodNames[i], bindingFlags) != null;
+
+                        // Check base type for target methods
+                        if (!targetMethodFound)
+                        {
+                            var componentBaseType = componentInstanceType.BaseType;
+                            if (componentBaseType != null)
+                                targetMethodFound = componentBaseType.GetMethod(targetMethodNames[i], bindingFlags) != null;
+                        }
```

Comment:
Add a Utils method to recursively check base types

Reply:
Added to ComponentUtils as "MethodFoundInBaseType"